web/skins/classic/includes/functions.php

```diff
@@ -78,6 +76,20 @@ function xhtmlHeaders( $file, $title )
   <script type="text/javascript" src="tools/mootools/mootools-core.js"></script>
   <script type="text/javascript" src="tools/mootools/mootools-more.js"></script>
   <script type="text/javascript" src="js/mootools.ext.js"></script>
+  <script type="text/javascript" src="skins/<?php echo $skin; ?>/js/jquery-1.11.3.js"></script>
```

Comment:
Instead of referencing jquery-1.11.3.js here, can we instead reference the symlink jquery.js which, today, points to jquery-1.11.3? Should make future upgrades of jquery easier.